articles/azure/azs-how-create-vm-image-powershell.md

```diff
@@ -0,0 +1,171 @@
+---
+title: How to create a custom image from a VM on Azure Stack - PowerShell | UKCloud Ltd
+description: Provides details on how to create a custom image from a VM using Powershell
+services: 
+author: Daniel Brennand
+reviewer:
+lastreviewed:
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a custom image from a VM - PowerShell
+toc_fullpath: Users/How To/azs-how-create-vm-image-powershell.md
+toc_mdlink: azs-how-create-vm-image-powershell.md
+---
+
+# How to create a custom image from a VM on Azure Stack - PowerShell
+
+## Overview
+
+An image resource can be created from a generalised virtual machine (VM) that is stored as either a managed disk or an unmanaged disk in a storage account. The image can then be used to create multiple VMs.
+
+This article will explain how to create a custom image from a VM on Azure Stack, which you can then use to deploy other VMs.
+
+## Prerequisites
+
+To complete the steps in this article, you must have appropriate access to a subscription in the **Azure Stack** portal.
+
+## [1. Generalise your VM](#tab/tabid-1)
+
+> [!WARNING]
+> Once a VM has been generalised, you **can not** log back into it.
+
+## [2. Creating the image - PowerShell](#tab/tabid-2)
```

Comment:
Now you've moved the portal steps into a different article, you don't need to include the - PowerShell bit in the heading here.

May also be able to simplify the tabs, but I can't tell from the markdown 😉